vectordatasource/transform.py

```diff
@@ -9472,9 +9484,33 @@ def apply_disputed_boundary_viewpoints(ctx):
         # we want to apply disputes to already generally-unrecognised borders
         # too, as this allows for multi-level fallback from one viewpoint
         # possibly through several others before reaching the default.
-        elif (kind.startswith('unrecognized_') and
-              kind[len('unrecognized_'):] in _BOUNDARY_KINDS):
-            boundaries.append((shape, props, fid))
+        elif kind.startswith('unrecognized_'):
+            if kind[len('unrecognized_'):] in _BOUNDARY_KINDS:
+                boundaries.append((shape, props, fid))
+
+        # this covers the disputed_reference_line case.
+        # unpack all the viewpoints from disputed_by, and make them all unrecognized
+        # TODO what do I need to do with geometry here?
+        elif kind == 'disputed_reference_line':
+            disputed_by = props.get('disputed_by')
+            if disputed_by:
+                for country in _list_of_countries(disputed_by):
+                    props['kind:' + country] = 'unrecognized_disputed_reference_line'
```

Comment:
Modulo other PR comments and IRL discussion update `unrecognized_disputed_reference_line` to `unrecognized_disputed`?

Reply:
Both of these are now valid.